src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -9,8 +9,7 @@ import {
   clearSearch,
   searchInputChange,
   selectInstitution,
-  searchSchools,
-  setCannotFindSchool
+  searchSchools
```

Comment:
Does anyone else have strong feelings on enforcing dangling commas to reduce diff noise like this? I used to hate them but have recently become a convert.

Reply:
I don't mind either way. I wish we could enforce alphabetization.